enterprise/server/test/integration/workflow/workflow_test.go

```diff
@@ -46,9 +52,39 @@ actions:
   - name: "Test action"
     triggers: { push: { branches: [ master ] } }
     bazel_commands: [ "build //:nop" ]
-`,
+` + osStr + archStr,
 	}
+
+	return fileNameToFileContentsMap
+}
+
+// repoContentsSlowScript simulates a test repo with the config files required to run a workflow
+// It sets up a slow sample script that takes a while to run so the CI runner does not return immediately,
+// giving tests that need to modify the workflow (Ex. for testing cancellation) time to complete
+func repoContentsSlowScript() map[string]string {
```

Comment:
nit: `repoContentsWithSlowBazelTest` might be a bit clearer

Reply:
Done